openapi.yaml

```diff
@@ -209,9 +438,324 @@ definitions:
+  playerInGameResource:
+    properties:
+      id:
+        $ref: '#/definitions/playerId'
+      type:
+        $ref: '#/definitions/playerType'
+      attributes:
+        type: object
+        properties:
+          memberName:
+            $ref: '#/definitions/memberName'
+          memberLevel:
+            $ref: '#/definitions/memberLevel'
+          memberExpAboveLevel:
+            $ref: '#/definitions/memberExpAboveLevel'
+          playerBet:
+            $ref: '#/definitions/playerBet'
+          playerStatus:
+            $ref: '#/definitions/playerStatus'
+          playerCards:
+            $ref: '#/definitions/cards'
+  playerInGameResult:
+    properties:
+      data:
+        $ref: '#/definitions/playerInGameResource'
+  playerInGameResults:
+    properties:
+      data:
+        type: array
+        items:
+          $ref: '#/definitions/playerInGameResource'
+  memberLevel:
+    type: integer
+    minimum: 1
+    description: Player's level in the system.
+    example: 20
+  memberName:
+    type: string
+    example: Jack
+  playerPassword:
+    type: string
+    example:  hunter2
+    format: password
+  memberPatchBody:
+    properties:
+      data:
+        type: object
+        properties:
+          id:
+            $ref: '#/definitions/playerId'
+          type:
+            $ref: '#/definitions/playerType'
+          attributes:
+            type: object
+            properties:
+              memberName:
+                $ref: '#/definitions/memberName'
+              playerEmail:
+                $ref: '#/definitions/playerEmail'
+              playerPassword:
+                $ref: '#/definitions/playerPassword'
+              memberLevel:
+                $ref: '#/definitions/memberLevel'
+              memberExpAboveLevel:
+                $ref: '#/definitions/memberExpAboveLevel'
+              gameIds:
+                $ref: '#/definitions/gameIds'
+            additionalProperties: false
+        required: 
+          - id
+          - type
+        additionalProperties: false
+  memberPostBody:
+    properties:
+      data:
+        type: object
+        properties:
+          type:
+            $ref: '#/definitions/playerType'
+          attributes:
+            type: object
+            properties:
+              memberName:
+                $ref: '#/definitions/memberName'
+              playerEmail:
+                $ref: '#/definitions/playerEmail'
+              playerPassword:
+                $ref: '#/definitions/playerPassword'
+            required:
+              - memberName
+              - playerEmail
+              - playerPassword
+            additionalProperties: false
+        required: 
+          - type
+          - attributes
+        additionalProperties: false
+    required: 
+      - data
+    additionalProperties: false 
+  memberResource:
+    properties:
+      id:
+        $ref: '#/definitions/playerId'
+      type:
+        $ref: '#/definitions/playerType'
```

Comment:
The `member` resource can't have the same `type` as the `player` resource. Otherwise, there could exist two resources with the same `type` and same `id` which would violate the jsonapi spec. You will need to have a different `type` if you want to use the same `id`.